packages/tasks/src/model-libraries.ts

```diff
@@ -406,6 +406,14 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		filter: false,
 		countDownloads: `path:"mlc-chat-config.json"`,
 	},
+	model2vec: {
+		prettyLabel: "Model2Vec",
+		repoName: "model2vec",
+		repoUrl: "https://github.com/MinishLab/model2vec",
+		snippets: snippets.model2vec,
+		filter: false,
+		countDownloads: `path_extension:"safetensors"`,
```

Comment:
The repos also contain a `config.json` so downloads should be computed by default. Would that work?

Reply:
Let me double-check whether the `config.json` is actually loaded, or if it's only pushed.

Reply:
Looks like it's indeed used. Let me remove this line for simplicity.